OtripleS.Web.Api/Services/CalendarEntryAttachments/CalendarEntryAttachmentService.Exceptions.cs

```diff
@@ -11,10 +11,12 @@
 
 namespace OtripleS.Web.Api.Services.CalendarEntryAttachments
 {
+    using System.Linq;
+
     public partial class CalendarEntryAttachmentService
     {
         private delegate ValueTask<CalendarEntryAttachment> ReturningCalendarEntryAttachmentFunction();
-
+        private delegate IQueryable<CalendarEntryAttachment> ReturningCalendarEntryAttachmentsFunction();
         private async ValueTask<CalendarEntryAttachment> TryCatch(
```

Comment:
Separate the two-liner code blocks by a new empty line from everything before and after